src/components/Button.js

```diff
@@ -58,7 +56,8 @@ class Button extends React.Component {
   };
 
   render () {
-    const styles = this.styles();
+    const theme = StyleUtils.mergeTheme(this.props.theme, this.props.primaryColor);
```

Comment:
Yes, that's the idea 😄 

See https://github.com/mxenabled/mx-react-components/pull/594/files#diff-734f11edfaa69e2a96b8cb8f7f06fb6bR34

And https://github.com/mxenabled/mx-react-components/pull/594/files#diff-ec85bd453c96420d8df62f4d96bd694aR33